app/javascript/widget/App.vue

```diff
@@ -8,7 +8,7 @@ import availabilityMixin from 'widget/mixins/availability';
 import { getLocale } from './helpers/urlParamsHelper';
 import { isEmptyObject } from 'widget/helpers/utils';
 import Spinner from 'shared/components/Spinner.vue';
-import routerMixin from './mixins/routerMixin';
+import { useRouterHelper } from 'widget/composables/useRouterHelper';
```

Comment:
We can just make a composable 'useReplaceRoute' instead of `useRouterHelper` that exposes the entire fn

Reply:
Resolved in https://github.com/chatwoot/chatwoot/pull/9941/commits/a0153daca413e55b9bfd86483624cac82e8b9eaf